Formula/rbenv.rb

```diff
@@ -24,19 +24,26 @@ class Rbenv < Formula
 
   def install
     inreplace "libexec/rbenv" do |s|
-      s.gsub! '"${BASH_SOURCE%/*}"/../libexec', libexec
+      # TODO: The following line can be removed in the next release.
+      # rbenv/rbenv/pull/1428 (`brew audit` doesn't like URLs of merged PRs.)
```

Comment:
Hi @SMillerDev

Reply:
yeah, it is kind of concern that I have.